TileStache/Goodies/VecTiles/transform.py

```diff
@@ -943,6 +945,10 @@ def exterior_boundaries(feature_layers, base_layer,
     """
     layer = None
 
+    # don't start processing until the start zoom
+    if zoom < start_zoom:
+        return layer
+
```

Comment:
If I understand correctly, maybe add a sanity check in here first to account for `start_zoom` not being set, and assume that means process?

```python
if start_zoom is None:
    start_zoom = 0
```